app/Observers/CommentObserver.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Observers;
+
+use App\Models\Comment;
+use App\Notifications\CommentNotification;
+use Illuminate\Support\Facades\Auth;
+
+class CommentObserver
+{
+    public function saved(Comment $comment)
+    {
+        if(Auth::check())
+        {
+            $booking = $comment->booking()->first();
+            $reviewers = $booking->reviewers()->get();
+            foreach ($reviewers as $reviewer) {
+                $reviewer->notify(new CommentNotification($comment));
```

Comment:
probably a good idea to pass the booking to the comment for the context info mentioned above

Reply:
done